distbench_utils.cc

```diff
@@ -302,10 +302,10 @@ struct rusage DoGetRusage() {
 }
 
 std::string GetNamedSettingString(
-    const distbench::ProtocolDriverOptions &opts,
+    const ::google::protobuf::RepeatedPtrField<distbench::NamedSetting> &opts,
```

Comment:
Maybe we should rename the argument to settings since the type has changed.

Reply:
Renamed.